scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl

```diff
@@ -1003,19 +1019,19 @@ task ClinvarSignificance {
         risk factor
         uncertain significance' > expected_clinvar_classes.csv
 
-        comm -23 <(sort bq_clinvar_classes.csv) expected_clinvar_classes.csv > missing_clinvar_classes.csv
+        comm -23 <(sort bq_clinvar_classes.csv) expected_clinvar_classes.csv > missing_clinvar_classes.txt
```

Comment:
Maybe the "csv" args to `comm` should be "txt"s as well since it's otherwise a bit confusing how the `comm`ing of two "csv"s begets a "txt".